toolbox/db/db_get.m

```diff
@@ -219,6 +219,48 @@
             varargout{1} = sAnatFiles;
         end
 
+%% ==== ANATOMY FILE ====
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', FileIDs)
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', FileNames)
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', CondQuery)
+    case 'AnatomyFile'
+        iFiles = args{1};
+        fields = '*';                              
+        templateStruct = db_template('AnatomyFile');
+        
+        if ischar(iFiles)
+            iFiles = {iFiles};
+        end
+        
+        if ischar(iFiles)
+            iFiles = {iFiles};
+        elseif isstruct(iFiles)
+            condQuery = CondQuery;           
+        end       
+        
+        
+        nFiles = length(iFiles);
+        sFiles = repmat(templateStruct, 1, nFiles);
+        
+        for i = 1:nFiles
+            if iscell(iFiles)
+                condQuery.FileName = iFiles{i};
+            else
+                condQuery.Id = iFiles(i);
+            end
+            sFiles(i) = sql_query(sqlConn, 'select', 'AnatomyFile', fields, condQuery);
+        end
+
+        varargout{1} = sFiles;
+        
+%% ==== SURFACE FILE ====
+    % Usage : [sSubject, iSubject, iSurface] = bst_get('SurfaceFile', SurfaceFile)
+    case 'SurfaceFile'
```

Comment:
This should be in bst_get. bst_get('SurfaceFile') still hasn't been refactored and since this was a public API we need to leave it functional.

Reply:
db_get('SurfaceFile') was refactored in 5395ff8
It is not optimal as it uses bst_get('ProtocolSubjects'). Thus the functions using it will be changed to db_get('SurfaceFile')
For legacy the output iSurface was kept as the index of the sSubject.Surface. Or should it be the index of the AnatomyFile table?

Reply:
Sounds good to me like this, thanks!

Reply:
I'm still not 100% convinced of the benefit of maintaining both db_get('SurfaceFile') and bst_get('SurfaceFile') with almost the same functionality, but if you think it's useful we can keep them both.

Reply:
Yeah, it was kind of redundant. Now bst_get('SurfaceFile') only calls db_get() functions. However, it's preferable to use db_get('AnatomyFile') when it's enough as it makes less reads in the DB.  Solved in 3834c65